src/helm/benchmark/run_specs.py

```diff
@@ -2536,7 +2542,20 @@ def construct_run_specs(spec: ObjectSpec) -> List[RunSpec]:
         raise ValueError(f"Unknown run spec name: {name}")
 
     # Peel off the run expanders (e.g., model)
-    expanders = [RUN_EXPANDERS[key](value) for key, value in args.items() if key in RUN_EXPANDERS]  # type: ignore
+    # DEPRECATED: Support for the old model name format
+    # All users should be using the model_deployment keyword argument instead.
+    # TODO: Remove this once we've migrated all the configs
+    expanders: List[RunExpander] = []
+    if args.get("model_deployment", None) is not None:
+        expanders.append(ModelDeploymentRunExpander(args["model_deployment"]))
+        args.pop("model_deployment")
+    elif args.get("model", None) is not None:
+        hlog("WARNING: Using deprecated model name format. Please use model_deployment instead.")
```

Comment:
Don't mark this as deprecated - this is still the workflow recommended by the tutorial!

Reply:
Well shouldn't we raise a warning? We need to changer the tutorial also.

Reply:
Do we actually want to deprecate `model=...` in favor of `model_deployments?

Remember that `model_deployments=text` does not actually exist, so we still need `model=text` to work... unless we get rid of `RunExpanders` entirely... which I have been suggesting to @percyliang for a while...